src/PlayBack/Slider.tsx

```diff
@@ -0,0 +1,105 @@
+import { FC, useMemo, useRef } from 'react';
+
+import { getLeft, getPercentage, getValue, throttled } from './playbackUtil';
+import { SliderContainer, SliderBar, SliderStyle, ThumbStyle } from './playbackStyle';
+import HighlightSections from './HighlightSections';
+import { HighlightsProp } from '../types/global';
+
+interface SliderProps {
+    value: number;
+    max: number;
+    highlights: HighlightsProp[];
+    onChange(index: number): void;
+}
+
+const Slider: FC<SliderProps> = ({ value, max, highlights, onChange }) => {
+    const initialPercentage = getPercentage(value, max);
+
+    const range = useMemo(() => {
+        let minRange = 0;
+        let maxRange = max;
+        if (highlights && highlights.length > 0 && highlights[0].playback) {
+            minRange = highlights[0].start;
+            maxRange = highlights[0].end;
+        }
+        return { min: minRange, max: maxRange };
+    }, [max, highlights]);
+
+    const sliderRef = useRef<HTMLDivElement>(null);
+    const thumbRef = useRef<HTMLDivElement>(null);
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
You can remove this comment. If it still causes a lint error, I believe you can change the following function explicitly return undefined:

```jsx
    const mouseMoveHandler = useRef(() => undefined);
```